erpnext_com/www/docs/user/manual/en/setting-up/data/personal-data-deletion.md

```diff
@@ -0,0 +1,46 @@
+<!--add breadcrumbs-->
+
+# Personal Data Deletion
+
+Personal data deletion tool enables a user to delete all the personal data a user has generated while using ERPNext.
+
+## Request to Delete Data Webform
```

Comment:
rename this as 1. How to request deleting your data or something similar. remove the next heading. continue these as steps. Kindly refer to any manual page from selling, buying, stock etc to see how the pages are structured, this needs to be consistent with that

Reply:
added proper indexing